console/encrypter.py

```diff
@@ -0,0 +1,88 @@
+from cryptography.hazmat.backends.openssl.backend import backend
+from cryptography.hazmat.primitives import serialization, hashes
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.primitives.serialization import Encoding, PublicFormat
+
+import base64
+import os
+from console import settings
+import jwt
+
+KID = 'SDE'
+
+
+class Encrypter (object):
+    def __init__(self):
+        private_key_bytes = self._to_bytes(settings.EQ_PRIVATE_KEY)
```

Comment:
See http://flask.pocoo.org/docs/0.12/config/#configuring-from-files for more info